frontend/src/component/admin/license/License.tsx

```diff
@@ -0,0 +1,140 @@
+
+        try {
+            await updateLicenseKey(token);
+            setToastData({
+                title: 'License key updated',
+                type: 'success',
+            });
+            refetchLicense();
+            reCheckLicense ();
+        } catch (error: unknown) {
+            setToastApiError(formatUnknownError(error));
+        }
+    };
+
+    return (
+        <PageContent header={<PageHeader title='Unleash Enterprise License' />}>
+            <StyledBox>
+                {license.token ? <div>
+                    <StyledDataCollectionPropertyRow>
+                        <StyledPropertyName>Customer</StyledPropertyName>
+                        <StyledPropertyDetails>
+                            {license?.customer}
+                        </StyledPropertyDetails>
+                    </StyledDataCollectionPropertyRow>
+                    <StyledDataCollectionPropertyRow>
+                        <StyledPropertyName>Plan</StyledPropertyName>
+                        <StyledPropertyDetails>
+                            {license?.plan}
+                        </StyledPropertyDetails>
+                    </StyledDataCollectionPropertyRow>
+                    <StyledDataCollectionPropertyRow>
+                        <StyledPropertyName>Seats</StyledPropertyName>
+                        <StyledPropertyDetails>
+                            {license?.seats}
+                        </StyledPropertyDetails>
+                    </StyledDataCollectionPropertyRow>
+                    <StyledDataCollectionPropertyRow>
+                        <StyledPropertyName>Expire at</StyledPropertyName>
+                        <StyledPropertyDetails>
+                            {formatDateYMD(license.expireAt, locationSettings.locale)}
+                        </StyledPropertyDetails>
+                    </StyledDataCollectionPropertyRow>
+                </div>: <p>You do not have a registered Unleash Enterprise License.</p>}
+                
+
+                <form onSubmit={onSubmit}>
+                    <TextField
+                        onChange={updateToken}
+                        label='New License Key'
+                        name='licenseKey'
+                        value={token}
+                        style={{ width: '100%' }}
+                        variant='outlined'
+                        size='small'
+                        multiline
+                        rows={6}
+                        required
+                    />
+                    <br /><br />
+                
+                    <Grid container spacing={3}>
+                        <Grid item md={5}>
+                            <Button
+                                variant='contained'
+                                color='primary'
+                                type='submit'
+                                disabled={loading}
+                            >
+                                Update License Key
```

Comment:
I think this would be more consistent with what we usually do.
```suggestion
                                Update license key
```